package.json

```diff
@@ -92,5 +92,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "engines": {
+    "node": ">=22.11.0"
```

Comment:
Would it be wrong to use an exact match here too?

Reply:
I was thinking we could do that, but any future developer would get stuck with newer versions of NodeJS (for example patch or minor). Perhaps that's a reasonable trade-off to make. Another thing to think about is that we probably do want to be testing on newer versions, so it might be too much friction to lock it down completely.